<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileDialog</class>
 <widget class="QDialog" name="ProfileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="buttonSave">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>130</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelImage">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(99, 99, 99);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>No image</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonImage">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>30</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string>Load image</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>20</width>
     <height>361</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>621</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="labelDescription">
      <property name="text">
       <string>Description:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEditDescription"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>351</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelManufacturer">
      <property name="text">
       <string>Manufacturer:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditManufacturer">
      <property name="maxLength">
       <number>12</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelType">
      <property name="text">
       <string>Type:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditType">
      <property name="maxLength">
       <number>12</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelModel">
      <property name="text">
       <string>Model:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEditModel">
      <property name="maxLength">
       <number>10</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="labelInfoToUser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>621</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Label info to user</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
